saleor/settings.py

```diff
@@ -463,7 +463,16 @@ def get_url_from_env(name, *, schemes=None) -> Optional[str]:
 # Following the recommendation of https://tools.ietf.org/html/rfc5322#section-2.1.1
 DEFAULT_MAX_EMAIL_DISPLAY_NAME_LENGTH = 78
 
-COUNTRIES_OVERRIDE = {"EU": "European Union"}
+COUNTRIES_OVERRIDE = {
+    "EU": "European Union",
+    "XK": {
```

Comment:
Should we also add at least one test for any Address mutation with `Kosovo` as a country?